### DISCUSSION TITLE
guide/drivers/quark

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#126] [Announcements]
# guide/drivers/quark

Due to the limitation of Quark network disk, the download must carry cookies, so only the proxy function in this program can be used for transfer. (You can use other machines to...

https://alist.nn.ci/guide/drivers/quark.html

**Comment by mavericklou** [comment]
使用夸克客户端下载速度可超过20MB/s，而用webdav映射后，下载速度为2MB/s，导致码率稍微高一点就造成卡顿，是什么原因？

**Reply by mavericklou** [reply]
_UP_D_=pc

**Reply by mavericklou** [reply]
cookies里面有这个属性，是不是网页版限速？还是大文件限速？

**Comment by xinghaix** [comment]
挂载夸克后，现在只能下载小文件了，大视频超过1G不能下载，而且不能在线播放视频了
